client/scraper/dropbox_metadata.py

```diff
@@ -23,14 +26,31 @@ def get_download_url(self, url):
         return urlunsplit((scheme, netloc, path, new_query_string, fragment))
 
     def parse_content(self, response):
-        self.generic_parse_content(response)
-
-        file_list = collections.OrderedDict()
-        file_list[FieldKeyword.COUNT] = 1
-        file_list[FieldKeyword.DATA] = [
-            {
-                FieldKeyword.URL: self.get_download_url(response.request_url),
-                FieldKeyword.TYPE: None
-            }]
-
-        self.prop_map[FieldKeyword.FILES] = file_list
+        soup = BeautifulSoup(response.content)
+        title = self.get_title(soup)
+
+        # TODO: currently unsure of how to tell if you have landed on a page that has 404ed since dropbox always returns 200 status
+        # the below is a hack to get around this problem, will need to clean this up once we've figured out how to resolve this issue
+        if title == 'Dropbox - Error':
+            self.prop_map[FieldKeyword.STATUS] = StatusCode.NOT_FOUND
```

Comment:
I have no clue if it always means 404, but we shouldn't be returning 200 status with a broken link. so this is a shorterm fix